src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,126 @@
 import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
 import './TodoList.scss';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.Component {
+  state = {
+    titleFilter: '',
+    statusFilter: '',
+  };
+
+  render() {
+    const { titleFilter, statusFilter } = this.state;
+    const {
+      todos,
+      onUserIdSelected,
+      selectedUserId, 
+    } = this.props;
+
+    const todosToDisplay = todos
+      .filter(todo => todo.title)
+        .filter(todo => todo.title.includes(titleFilter))
+          .filter(todo => {
```

Comment:
```suggestion
      .filter(todo => todo.title)
      .filter(todo => todo.title.includes(titleFilter))
      .filter(todo => {
```
Chaining methods should be written with the same indentation